docs/guide/built-in-rules-nested.md

```diff
@@ -31,17 +31,17 @@ The output of `$errors` will be:
     'author.age' => ['Value must be no less than 18.'],
 ];
 ```
-In this example an additional instance of `Nested` rule is used for every relation. The other ways of configurations are 
-possible, they are described further on.
+In this example, an additional instance of the `Nested` rule is used for each relation. Other types of configuration
```

Comment:
"every" is used here intentionally to avoid confusion with `Each` rule.

Reply:
> Every is similar to each in that both words are used to reference groups of items. While each can be used for groups of two or more, every refers to groups of at least three items. Additionally, every refers to the collective whole, not just the individuals.

https://www.grammarly.com/blog/each-and-every/